2024-07-17 19:50:35,355 : INFO : root : Loading config.
2024-07-17 19:50:35,357 : INFO : YamlMultifileConfigLoader : Machine config file #1: config.yaml
2024-07-17 19:50:35,357 : INFO : ConfigProcessor : Loading config from cache: /var/folders/sv/d29kcl5x50v7vt1n6xb5jsd40000gn/T/c166cef8060ffee16a35821bf698da85.mpf_cache
2024-07-17 19:50:35,384 : INFO : ConfigProcessor : Loading config from cache: /var/folders/sv/d29kcl5x50v7vt1n6xb5jsd40000gn/T/6b31c5ac6ecbad948994a8204ea6b05b.mpf_cache
2024-07-17 19:50:35,384 : INFO : ConfigProcessor : Loading config from cache: /var/folders/sv/d29kcl5x50v7vt1n6xb5jsd40000gn/T/fe612d1e66c8a96c19ba449d3748eeac.mpf_cache
2024-07-17 19:50:35,385 : INFO : ConfigProcessor : Loading config from cache: /var/folders/sv/d29kcl5x50v7vt1n6xb5jsd40000gn/T/6b31c5ac6ecbad948994a8204ea6b05b.mpf_cache
2024-07-17 19:50:35,385 : INFO : ConfigProcessor : Loading config from cache: /var/folders/sv/d29kcl5x50v7vt1n6xb5jsd40000gn/T/8765227693ecd961e0504502a81255e0.mpf_cache
2024-07-17 19:50:35,385 : INFO : ConfigProcessor : Loading config from cache: /var/folders/sv/d29kcl5x50v7vt1n6xb5jsd40000gn/T/b247728c74b4d81bcb2825e2b07062a7.mpf_cache
2024-07-17 19:50:35,385 : INFO : ConfigProcessor : Loading config from cache: /var/folders/sv/d29kcl5x50v7vt1n6xb5jsd40000gn/T/f240f94f0b5d0f37ec4fd7571275a101.mpf_cache
2024-07-17 19:50:35,386 : INFO : ConfigProcessor : Loading config from cache: /var/folders/sv/d29kcl5x50v7vt1n6xb5jsd40000gn/T/c4409acfdf3ec94666e0adef48d55400.mpf_cache
2024-07-17 19:50:35,386 : INFO : ConfigProcessor : Loading config from cache: /var/folders/sv/d29kcl5x50v7vt1n6xb5jsd40000gn/T/6a2f50ed892485825cc04dc0a68eab9f.mpf_cache
2024-07-17 19:50:35,386 : INFO : ConfigProcessor : Loading config from cache: /var/folders/sv/d29kcl5x50v7vt1n6xb5jsd40000gn/T/d58c58eb9fd1a2dc27f8acd255fb3761.mpf_cache
2024-07-17 19:50:35,386 : INFO : ConfigProcessor : Loading config from cache: /var/folders/sv/d29kcl5x50v7vt1n6xb5jsd40000gn/T/07adfd310b3ef0e03c5cbc91527abb6c.mpf_cache
2024-07-17 19:50:35,386 : INFO : ConfigProcessor : Loading config from cache: /var/folders/sv/d29kcl5x50v7vt1n6xb5jsd40000gn/T/12bd5247c3bf989b919364bdb7829769.mpf_cache
2024-07-17 19:50:35,387 : INFO : ConfigProcessor : Loading config from cache: /var/folders/sv/d29kcl5x50v7vt1n6xb5jsd40000gn/T/362e08f51a1a1b7a0809dd52f9c83f83.mpf_cache
2024-07-17 19:50:35,387 : INFO : ConfigProcessor : Loading config from cache: /var/folders/sv/d29kcl5x50v7vt1n6xb5jsd40000gn/T/e45e3bd39557ed8d0881194a7afab8f7.mpf_cache
2024-07-17 19:50:35,387 : INFO : ConfigProcessor : Loading config from cache: /var/folders/sv/d29kcl5x50v7vt1n6xb5jsd40000gn/T/331d8d1a1bc190cae8c13495e1db372f.mpf_cache
2024-07-17 19:50:35,387 : INFO : ConfigProcessor : Loading config from cache: /var/folders/sv/d29kcl5x50v7vt1n6xb5jsd40000gn/T/d17a9ff77386e6f1439d0802ace1de07.mpf_cache
2024-07-17 19:50:35,387 : INFO : ConfigProcessor : Loading config from cache: /var/folders/sv/d29kcl5x50v7vt1n6xb5jsd40000gn/T/280fd64455d0d243a54ab4db98fbccbd.mpf_cache
2024-07-17 19:50:35,387 : WARNING : ConfigProcessor : Config file in cache changed: /Users/nicktaseris/Desktop/Pinball/KOTH/modes/mission_select/config/mission_select.yaml
2024-07-17 19:50:35,387 : INFO : ConfigProcessor : Loading config from file /Users/nicktaseris/Desktop/Pinball/KOTH/modes/mission_select/config/mission_select.yaml.
2024-07-17 19:50:35,389 : INFO : ConfigProcessor : Loading config: /Users/nicktaseris/Desktop/Pinball/KOTH/modes/mission_select/config/mission_select.yaml
2024-07-17 19:50:35,389 : INFO : ConfigProcessor : Config file cache created: /var/folders/sv/d29kcl5x50v7vt1n6xb5jsd40000gn/T/0c0596ed8fbb40b6113901d21bdc0a39.mpf_cache
2024-07-17 19:50:35,390 : INFO : ConfigProcessor : Loading config from cache: /var/folders/sv/d29kcl5x50v7vt1n6xb5jsd40000gn/T/2982e2d98506102244e9fa0f437a9fca.mpf_cache
2024-07-17 19:50:35,390 : INFO : ConfigProcessor : Loading config from cache: /var/folders/sv/d29kcl5x50v7vt1n6xb5jsd40000gn/T/0fedc249e142fba0aaf46d9adc7061fa.mpf_cache
2024-07-17 19:50:35,390 : INFO : ConfigProcessor : Loading config from cache: /var/folders/sv/d29kcl5x50v7vt1n6xb5jsd40000gn/T/3048ed2d7ea14eee3db896f6760aa474.mpf_cache
2024-07-17 19:50:35,391 : INFO : Machine : Mission Pinball Framework Core Engine v0.80.0.dev1
2024-07-17 19:50:35,391 : INFO : Machine : Command line arguments: {'no_load_cache': False, 'create_config_cache': True, 'bcp': True, 'configfile': ['config.yaml'], 'force_assets_load': False, 'platform_integration_test': False, 'jsonlogging': False, 'logfile': 'logs/2024-07-17-19-50-35-mpf-Nicks-Air.lan.log', 'pause': False, 'production': False, 'text_ui': True, 'loglevel': 15, 'consoleloglevel': 20, 'force_platform': None, 'syslog_address': None, 'mc_file_name': None, 'no_sound': False, 'full_logfile_path': '/Users/nicktaseris/Desktop/Pinball/KOTH/logs/2024-07-17-19-50-35-mpf-Nicks-Air.lan.log'}
2024-07-17 19:50:35,391 : INFO : Machine : MPF path: /Users/nicktaseris/mpfenv/lib/python3.9/site-packages/mpf
2024-07-17 19:50:35,391 : INFO : Machine : Machine path: /Users/nicktaseris/Desktop/Pinball/KOTH
2024-07-17 19:50:35,391 : INFO : Machine : Platform: darwin
2024-07-17 19:50:35,391 : INFO : Machine : Python executable location: /Users/nicktaseris/mpfenv/bin/python3
2024-07-17 19:50:35,391 : INFO : Machine : Python version: 3.9.13 (64-bit)
2024-07-17 19:50:35,391 : INFO : Machine : Initializing MPF...
2024-07-17 19:50:35,419 : INFO : EventManager : Event: ======'machine_var_credits_string'====== Args={'value': 'FREE PLAY', 'prev_value': None, 'change': True}
2024-07-17 19:50:35,424 : INFO : EventManager : Event: ======'machine_var_master_volume'====== Args={'value': 0.5, 'prev_value': None, 'change': True}
2024-07-17 19:50:35,424 : INFO : EventManager : Event: ======'machine_var_player1_score'====== Args={'value': 175300, 'prev_value': None, 'change': True}
2024-07-17 19:50:35,425 : INFO : EventManager : Event: ======'machine_var_mpf_version'====== Args={'value': 'Mission Pinball Framework v0.80.0.dev1', 'prev_value': None, 'change': True}
2024-07-17 19:50:35,425 : INFO : EventManager : Event: ======'machine_var_mpf_extended_version'====== Args={'value': 'Mission Pinball Framework v0.80.0.dev1, Config version:6, Show version: 6, BCP version:1.1', 'prev_value': None, 'change': True}
2024-07-17 19:50:35,425 : INFO : EventManager : Event: ======'machine_var_python_version'====== Args={'value': '3.9.13', 'prev_value': None, 'change': True}
2024-07-17 19:50:35,433 : INFO : EventManager : Event: ======'machine_var_platform'====== Args={'value': 'macOS-12.1-arm64-arm-64bit', 'prev_value': None, 'change': True}
2024-07-17 19:50:35,434 : INFO : EventManager : Event: ======'machine_var_platform_system'====== Args={'value': 'Darwin', 'prev_value': None, 'change': True}
2024-07-17 19:50:35,434 : INFO : EventManager : Event: ======'machine_var_platform_release'====== Args={'value': '21.2.0', 'prev_value': None, 'change': True}
2024-07-17 19:50:35,434 : INFO : EventManager : Event: ======'machine_var_platform_version'====== Args={'value': 'Darwin Kernel Version 21.2.0: Sun Nov 28 20:29:10 PST 2021; root:xnu-8019.61.5~1/RELEASE_ARM64_T8101', 'prev_value': None, 'change': True}
2024-07-17 19:50:35,434 : INFO : EventManager : Event: ======'machine_var_platform_machine'====== Args={'value': 'arm64', 'prev_value': None, 'change': True}
2024-07-17 19:50:35,434 : INFO : EventManager : Event: ======'machine_var_log_file_path'====== Args={'value': '/Users/nicktaseris/Desktop/Pinball/KOTH/logs/2024-07-17-19-50-35-mpf-Nicks-Air.lan.log', 'prev_value': None, 'change': True}
2024-07-17 19:50:35,434 : INFO : EventManager : Event: ======'init_phase_1'====== Args={}
2024-07-17 19:50:35,457 : INFO : EventManager : Event: ======'drop_target_bank_strickland_bank_up'====== Args={}
2024-07-17 19:50:35,458 : INFO : EventManager : Event: ======'init_phase_2'====== Args={}
2024-07-17 19:50:35,458 : INFO : SwitchController : Found active switch: ('39', <Platform.SmartVirtual>) <switch.s_trough1>
2024-07-17 19:50:35,458 : INFO : SwitchController : Found active switch: ('40', <Platform.SmartVirtual>) <switch.s_trough2>
2024-07-17 19:50:35,458 : INFO : SwitchController : Found active switch: ('41', <Platform.SmartVirtual>) <switch.s_trough3>
2024-07-17 19:50:35,458 : INFO : SwitchController : Found active switch: ('42', <Platform.SmartVirtual>) <switch.s_trough4>
2024-07-17 19:50:35,458 : INFO : SwitchController : Found active switch: ('43', <Platform.SmartVirtual>) <switch.s_trough5>
2024-07-17 19:50:35,458 : INFO : SwitchController : Found active switch: ('44', <Platform.SmartVirtual>) <switch.s_trough6>
2024-07-17 19:50:35,460 : ERROR : asyncio : Exception in callback EventManager._queue_task_done(<Task finishe...boomhauer_1')>)
handle: <Handle EventManager._queue_task_done(<Task finishe...boomhauer_1')>)>
Traceback (most recent call last):
  File "/Users/nicktaseris/mpfenv/lib/python3.9/site-packages/mpf/config_players/light_player.py", line 122, in _convert_color
    color = RGBColor(color)
  File "/Users/nicktaseris/mpfenv/lib/python3.9/site-packages/mpf/core/rgb_color.py", line 177, in __init__
    self._color = RGBColor.string_to_rgb(color)
  File "/Users/nicktaseris/mpfenv/lib/python3.9/site-packages/mpf/core/rgb_color.py", line 420, in string_to_rgb
    raise ColorException("Invalid RGB string: {}".format(value))
mpf.core.rgb_color.ColorException: Invalid RGB string: l_boomhauer_1

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/asyncio/events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "/Users/nicktaseris/mpfenv/lib/python3.9/site-packages/mpf/core/events.py", line 795, in _queue_task_done
    future.result()
  File "/Users/nicktaseris/mpfenv/lib/python3.9/site-packages/mpf/core/events.py", line 708, in _run_handlers_sequential
    handler.callback(queue=queue, **merged_kwargs)
  File "/Users/nicktaseris/mpfenv/lib/python3.9/site-packages/mpf/core/mode_controller.py", line 97, in initialize_modes
    item.method(config=mode.config[item.config_section],
  File "/Users/nicktaseris/mpfenv/lib/python3.9/site-packages/mpf/core/config_player.py", line 150, in process_mode_config
    config = self.validate_config(config)
  File "/Users/nicktaseris/mpfenv/lib/python3.9/site-packages/mpf/core/config_player.py", line 113, in validate_config
    validated_config[event] = self.validate_config_entry(settings, event)
  File "/Users/nicktaseris/mpfenv/lib/python3.9/site-packages/mpf/config_players/device_config_player.py", line 56, in validate_config_entry
    raise e
  File "/Users/nicktaseris/mpfenv/lib/python3.9/site-packages/mpf/config_players/device_config_player.py", line 52, in validate_config_entry
    self._validate_config_item(device, device_settings))
  File "/Users/nicktaseris/mpfenv/lib/python3.9/site-packages/mpf/config_players/device_config_player.py", line 83, in _validate_config_item
    raise e
  File "/Users/nicktaseris/mpfenv/lib/python3.9/site-packages/mpf/config_players/device_config_player.py", line 80, in _validate_config_item
    device_settings = self._expand_device_config(device_settings)
  File "/Users/nicktaseris/mpfenv/lib/python3.9/site-packages/mpf/config_players/light_player.py", line 133, in _expand_device_config
    device_settings['color'] = self._convert_color(color)
  File "/Users/nicktaseris/mpfenv/lib/python3.9/site-packages/mpf/config_players/light_player.py", line 124, in _convert_color
    self.raise_config_error("Invalid color {}".format(color), 1, source_exception=e, context=context)
  File "/Users/nicktaseris/mpfenv/lib/python3.9/site-packages/mpf/core/logging.py", line 174, in raise_config_error
    raise ConfigFileError(msg, error_no, self.log.name if self.log else "", context, self._url_base) \
mpf.exceptions.config_file_error.ConfigFileError: Config File Error in light_player: Failed to load config player mission_beat_the_surfer_highlighted:lights with settings {'light': 'l_boomhauer_1'} >> Failed to expand entry "light". >> Invalid color l_boomhauer_1 Error Code: CFE-light_player-1 (https://missionpinball.org/logs)
2024-07-17 19:50:35,462 : INFO : Machine : Starting the main run loop.
2024-07-17 19:50:35,463 : INFO : Machine : Shutting down...
2024-07-17 19:50:35,463 : INFO : EventManager : Event: ======'shutdown'====== Args={}
2024-07-17 19:50:36,465 : ERROR : Machine : Runtime Exception
Traceback (most recent call last):
  File "/Users/nicktaseris/mpfenv/lib/python3.9/site-packages/mpf/config_players/light_player.py", line 122, in _convert_color
    color = RGBColor(color)
  File "/Users/nicktaseris/mpfenv/lib/python3.9/site-packages/mpf/core/rgb_color.py", line 177, in __init__
    self._color = RGBColor.string_to_rgb(color)
  File "/Users/nicktaseris/mpfenv/lib/python3.9/site-packages/mpf/core/rgb_color.py", line 420, in string_to_rgb
    raise ColorException("Invalid RGB string: {}".format(value))
mpf.core.rgb_color.ColorException: Invalid RGB string: l_boomhauer_1

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/nicktaseris/mpfenv/lib/python3.9/site-packages/mpf/core/machine.py", line 793, in _run_loop
    raise self._exception['exception']
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/asyncio/events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "/Users/nicktaseris/mpfenv/lib/python3.9/site-packages/mpf/core/events.py", line 795, in _queue_task_done
    future.result()
  File "/Users/nicktaseris/mpfenv/lib/python3.9/site-packages/mpf/core/events.py", line 708, in _run_handlers_sequential
    handler.callback(queue=queue, **merged_kwargs)
  File "/Users/nicktaseris/mpfenv/lib/python3.9/site-packages/mpf/core/mode_controller.py", line 97, in initialize_modes
    item.method(config=mode.config[item.config_section],
  File "/Users/nicktaseris/mpfenv/lib/python3.9/site-packages/mpf/core/config_player.py", line 150, in process_mode_config
    config = self.validate_config(config)
  File "/Users/nicktaseris/mpfenv/lib/python3.9/site-packages/mpf/core/config_player.py", line 113, in validate_config
    validated_config[event] = self.validate_config_entry(settings, event)
  File "/Users/nicktaseris/mpfenv/lib/python3.9/site-packages/mpf/config_players/device_config_player.py", line 56, in validate_config_entry
    raise e
  File "/Users/nicktaseris/mpfenv/lib/python3.9/site-packages/mpf/config_players/device_config_player.py", line 52, in validate_config_entry
    self._validate_config_item(device, device_settings))
  File "/Users/nicktaseris/mpfenv/lib/python3.9/site-packages/mpf/config_players/device_config_player.py", line 83, in _validate_config_item
    raise e
  File "/Users/nicktaseris/mpfenv/lib/python3.9/site-packages/mpf/config_players/device_config_player.py", line 80, in _validate_config_item
    device_settings = self._expand_device_config(device_settings)
  File "/Users/nicktaseris/mpfenv/lib/python3.9/site-packages/mpf/config_players/light_player.py", line 133, in _expand_device_config
    device_settings['color'] = self._convert_color(color)
  File "/Users/nicktaseris/mpfenv/lib/python3.9/site-packages/mpf/config_players/light_player.py", line 124, in _convert_color
    self.raise_config_error("Invalid color {}".format(color), 1, source_exception=e, context=context)
  File "/Users/nicktaseris/mpfenv/lib/python3.9/site-packages/mpf/core/logging.py", line 174, in raise_config_error
    raise ConfigFileError(msg, error_no, self.log.name if self.log else "", context, self._url_base) \
mpf.exceptions.config_file_error.ConfigFileError: Config File Error in light_player: Failed to load config player mission_beat_the_surfer_highlighted:lights with settings {'light': 'l_boomhauer_1'} >> Failed to expand entry "light". >> Invalid color l_boomhauer_1 Error Code: CFE-light_player-1 (https://missionpinball.org/logs)
2024-07-17 19:50:36,547 : INFO : crash_reporter : Will not report crash as "report_crashes" mode is "never"
2024-07-17 19:50:36,547 : INFO : root : MPF run loop ended.
